rest/_examples/account.go

```diff
@@ -56,4 +57,14 @@ func main() {
 		b, _ := json.MarshalIndent(k, "", "  ")
 		fmt.Println(string(b))
 	}
+
+	key := account.APIKey{
+		Name:              "new-example",
+		IPWhitelist:       []string{"1.1.1.1", "2.2.2.2"},
+		IPWhitelistStrict: true,
+	}
+
+	if _, err := client.APIKeys.Create(&key); err != nil {
```

Comment:
@Zach-Johnson Thanks for adding this so quick! Is there a trick to enabling it (beyond the `api.SetDDIAPI()` decorator)?

I have the decorator set on my client, but while I see the `ip_whitelist` in the response from the server I do not see the field included in the request for either a `Create()` or `Update()`.

Relevant code:

```go
		fmt.Println(utils.PrettyPrint(k))
		if !exists {
			l.Infoln("        Creating")
			_, err = ns1.APIKeys.Create(k)
		} else if changed {
			l.Infoln("        Updating")
			_, err = ns1.APIKeys.Update(k)
		}
```

Logging output that shows the JSON dump of the `APIKey` object from the first line of that snippet and then my HTTP client's custom logging of the POST request and response. We can clearly see that I have the `IPWhitelist` field populated when I dump the key object, but it is then not included in the POST request body.

```
{
    "name": "security_user",
    "teams": [
        "5e2095f6808bc70061b249be"
    ],
    "permissions": {
        "dns": {
            "view_zones": false,
            "manage_zones": false,
            "zones_allow_by_default": false,
            "zones_deny": [],
            "zones_allow": []
        },
        "data": {
            "push_to_datafeeds": false,
            "manage_datasources": false,
            "manage_datafeeds": false
        },
        "account": {
            "manage_users": false,
            "manage_payment_methods": false,
            "manage_plan": false,
            "manage_teams": false,
            "manage_apikeys": false,
            "manage_account_settings": false,
            "view_activity_log": false,
            "view_invoices": false
        },
        "monitoring": {
            "manage_lists": false,
            "manage_jobs": false,
            "view_jobs": false
        },
        "security": {
            "manage_global_2fa": false
        },
        "dhcp": {
            "manage_dhcp": false,
            "view_dhcp": false
        },
        "ipam": {
            "manage_ipam": false,
            "view_ipam": false
        }
    },
    "ip_whitelist": [
        "1.1.1.1",
        "2.2.2.2",
        "3.3.3.3"
    ],
    "ip_whitelist_strict": true
}
2020/01/16 11:57:28         Creating
2020/01/16 11:57:28 DEBUG: client.(*HTTPClient).Do(git.soma.salesforce.com/ddi/ddisrv/http/client/client.go:177): Request:
    Method: PUT
    URL: https://localhost/v1/account/apikeys
    Proto: HTTP/1.1
    Headers:
        User-Agent:
            ns1-stuff-security/1.0
        X-Nsone-Key:
            YYY
    Trailer:
    Content Length: 577
    Host: localhost
    Form: map[]
    PostForm: map[]
    RemoteAddr: 
    RequestURI: 
    Body:
{"name":"security_user","teams":["5e2095f6808bc70061b249be"],"permissions":{"dns":{"view_zones":false,"manage_zones":false,"zones_allow_by_default":false,"zones_deny":[],"zones_allow":[]},"data":{"push_to_datafeeds":false,"manage_datasources":false,"manage_datafeeds":false},"account":{"manage_users":false,"manage_teams":false,"manage_apikeys":false,"manage_account_settings":false,"view_activity_log":false},"security":{"manage_global_2fa":false,"manage_active_directory":false},"dhcp":{"manage_dhcp":false,"view_dhcp":false},"ipam":{"manage_ipam":false,"view_ipam":false}}}
Response:
    Status: 200 OK
    Proto: HTTP/1.1
    Headers:
        X-Ratelimit-By:
            customer
        Pragma:
            no-cache
        X-Ratelimit-Period:
            1
        X-Content-Type-Options:
            nosniff
        Date:
            Thu, 16 Jan 2020 16:57:28 GMT
        Connection:
            keep-alive
        X-Ratelimit-Remaining:
            999
        Server:
            NSONE API v1
        X-Ratelimit-Limit:
            1000
        Cache-Control:
            no-cache
            private, max-age=0, no-cache, no-store
        X-Xss-Protection:
            1; mode=block
        Content-Type:
            application/json
        Expires:
            Thu, 16 Jan 2020 16:57:27 GMT
        X-Frame-Options:
            SAMEORIGIN
        Strict-Transport-Security:
            max-age=63072000; includeSubdomains; preload
    Trailer:
    Content Length: -1
    Uncompressed: true
    Body:
{"ip_whitelist_strict":false,"name":"security_user","key":"XXX","ip_whitelist":[],"last_access":null,"teams":["5e2095f6808bc70061b249be"],"id":"5e2095f8808bc70061b249e2","permissions":{"account":{"manage_account_settings":true,"manage_apikeys":true,"manage_users":true,"manage_teams":true,"view_activity_log":true},"ipam":{"view_ipam":false,"manage_ipam":false},"dns":{"zones_allow":[],"manage_zones":false,"zones_deny":[],"zones_allow_by_default":false,"view_zones":false},"dhcp":{"view_dhcp":false,"manage_dhcp":false},"security":{"manage_global_2fa":true,"manage_active_directory":true},"data":{"manage_datafeeds":false,"manage_datasources":false,"push_to_datafeeds":false}}}
```